lib/galaxy/exceptions/__init__.py

```diff
@@ -30,7 +30,7 @@ class MessageException(Exception):
     # Error code information embedded into API json responses.
     err_code: ErrorCode = error_codes_by_name["UNKNOWN"]
 
-    def __init__(self, err_msg=None, type="info", **extra_error_info):
+    def __init__(self, err_msg: str = "", type="info", **extra_error_info):
```

Comment:
Why did you make this changes ? `self.err_msg = err_msg or self.err_code.default_error_message` should prevent None being the err_msg attribute in any case, so then it seems like
```suggestion
    def __init__(self, err_msg: Optional[str] = None, type="info", **extra_error_info):
```
would be right type hint ?

Reply:
That was just me trying to make it blow up; `Optional[str]` with a default of None is totally fine.